### PR TITLE
Add the logic to upgrade ocs version

### PR DESCRIPTION
**Pull request by Gangadhar-ibm-lab**
Add the logic to upgrade ocs version

### CODE REVIEW

File: ocs_ci/ocs/ocs_upgrade.py [hunk 1]
```diff
@@ -595,58 +596,78 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
...
+        ):
+            clustername = config.ENV_DATA.get("cluster_name")
+            ocs_version = version.get_semantic_ocs_version_from_config()
+            cmd = f"ibmcloud ks cluster addon disable openshift-data-foundation --cluster {clustername} "
+            run_ibmcloud_cmd(cmd)
+            time.sleep(120)
+            cmd = (
+                f"ibmcloud ks cluster addon enable openshift-data-foundation --cluster {clustername} -f --version "
+                f"{ocs_version}.0 --param ocsUpgrade=true"
+            )
```
Comment (petr-balogh) [inline ocs_ci/ocs/ocs_upgrade.py:614]:
```suggestion
            cmd = f"ibmcloud ks cluster addon disable openshift-data-foundation --cluster {clustername} "
            run_ibmcloud_cmd(cmd)
            time.sleep(120)
            cmd = (
                f"ibmcloud ks cluster addon enable openshift-data-foundation --cluster {clustername} -f --version "
                f"{upgrade_version}.0 --param ocsUpgrade=true"
            )
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ mirror_openshift:
   password: "PASSWORD"
 ```
 
+#### vSphere UPI
```
Comment (petr-balogh) [inline docs/getting_started.md:39]: Please rebase as this should not be here as it is not your commit

**Comment by petr-balogh** [comment]
Verification triggered here:
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/14109/

**Review by petr-balogh — APPROVED** [review]

**Review by shylesh — APPROVED** [review]
LGTM

**Pull request merged**